modules/utxo-lib/examples/utxo/create-address-btg.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ *
+ * Create various types of bitcoin gold addresses. utxolib derives from bitcoinjs-lib.
+ *
+ */
+
+{
```

Comment:
why wrap in braces?

Reply:
I put them in as code blocks to get rid of the linting errors even though they don't actually run as part of the project was all.